hollow/src/main/java/com/netflix/hollow/api/consumer/fs/HollowFilesystemBlobRetriever.java

```diff
@@ -260,5 +347,37 @@ public File getFile() throws IOException {
 
             return path.toFile();
         }
+
+        @Override
+        public OptionalBlobPartInput getOptionalBlobPartInputs() throws IOException {
+            OptionalBlobPartInput remoteOptionalParts = remoteBlob.getOptionalBlobPartInputs();
+            if(remoteOptionalParts == null)
+                return null;
+
+            OptionalBlobPartInput localOptionalParts = new OptionalBlobPartInput();
+
+
+            for(Map.Entry<String, HollowBlobInput> entry : remoteOptionalParts.getInputsByPartName().entrySet()) {
```

Comment:
wonder if there'd be much to gain by doing these in parallel

Reply:
I think we don't want that -- we don't want to use up all the CPU on the consumer instances to do our delta updates -- taking a bit longer is preferable.